doc/source/WorkingWithThePythonScriptingAPI/ThePythonEditor/index.md

```diff
@@ -50,12 +50,12 @@ The specific shortcut for dragging depends on the source editor:
 ======================== ================= ====================== ============ ============================
 Element                  Source editor     Shortcut               Cursor       Example code result
 ======================== ================= ====================== ============ ============================
-Node(s)                  *Graph Editor*    Middle-click and drag  |nodes|      ``root['Sphere']``
-Plug                     *Node Editor*     Click and drag         |plug|       ``root['Sphere']['radius']``
-Plug value               *Node Editor*     Middle-click and drag  |values|     ``2.7``
-Object(s)                *Viewer*          Click and drag         |objects|    ``['/sphere']``
-Scene location(s)        *Hierarchy View*  Click and drag         |objects|    ``['/sphere']``
-Scene data               *Scene Inspector* Click and drag         |values|     ``V3f(1.5, 2.7, 3.9)``
+Node(s)                  Graph Editor      Middle-click and drag  |nodes|      ``root['Sphere']``
+Plug                     Node Editor       Click and drag         |plug|       ``root['Sphere']['radius']``
+Plug value               Node Editor       Middle-click and drag  |values|     ``2.7``
+Object(s)                Viewer            Click and drag         |objects|    ``['/sphere']``
+Scene location(s)        Hierarchy View    Click and drag         |objects|    ``['/sphere']``
+Scene data               Scene Inspector   Click and drag         |values|     ``V3f(1.5, 2.7, 3.9)``
```

Comment:
Is there something funky going on here with the white space?

Reply:
Have to maintain spacing so the table is read correctly.